spec/controllers/api/v1/rents_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+describe RentsController, type: :controller do
```

Comment:
It's ok.  But `type: :controller` is optional and i don't use it :smile:

Reply:
Done!